processor/transformprocessor/README.md

```diff
@@ -486,8 +596,8 @@ Supported aggregation functions are:
 
 Examples:
 
-- `aggregate_on_attributes("sum", ["attr1", "attr2"]) where name == "system.memory.usage"`
-- `aggregate_on_attributes("max") where name == "system.memory.usage"`
+- `aggregate_on_attributes("sum", ["attr1", "attr2"]) where metric.name == "system.memory.usage"`
+- `aggregate_on_attributes("max") where metric.name == "system.memory.usage"`
```

Comment:
The example ~10 lines below this needs the paths updated.

Reply:
Fixed, thank you!